habitat_baselines/common/base_trainer.py

```diff
@@ -18,7 +18,13 @@
     TensorboardWriter,
     get_writer,
 )
-from habitat_baselines.rl.ddppo.ddp_utils import SAVE_STATE, is_slurm_batch_job
+from habitat_baselines.rl.ddppo.ddp_utils import (
+    SAVE_STATE,
+    add_signal_handlers,
+    is_slurm_batch_job,
+    load_resume_state,
+    save_resume_state,
+)
```

Comment:
I am against importing those in both base_trainer and in ppo_trainer. Also against doing 
```
        if is_slurm_batch_job():
            add_signal_handlers()
```
In both eval and training. If both need to call it, let's put in in a helper method and call it from both.